Assets/HoloToolkit/ARCA/Scripts/SpatialSync/AnchorLocated.cs

```diff
@@ -1,43 +1,67 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
-
+// Licensed under the MIT License. See LICENSE in the project root for license information.
 using UnityEngine;
 using UnityEngine.XR.iOS;
```

Comment:
Does this need to be wrapped in `UNITY_IOS` to make sure it compiles for other platforms?